.travis.yml

```diff
@@ -228,6 +228,10 @@ script:
           echo -e '\052\052 FAILED -- MAKE';
           travis_terminate 1;
       fi;
+    - if test -n "$CHECKDOCS" && ! $make cmd-nits; then
+          echo -e '\052\052 FAILED -- MAKE CMD-NITS';
```

Comment:
Why the octal escapes?

Octal 52 is '*' which isn't special inside single quotes.

Reply:
But, the command itself may be echoed, so this allows us to find the actual output apart from the echoed echo command.

Reply:
Since we now use travis_terminate, this is less important than it used to be (just jump to the end of the log), but it still works as-is so it seems not necessary to change it.